docs/deps.md

```diff
@@ -0,0 +1 @@
+# Dependencies
```

Comment:
Fill in with actual documentation, I think we just include it directly in this PR.